lib/tasks/search.rake

```diff
@@ -18,8 +18,8 @@ namespace :search do
 
     Search::Cluster.delete_indexes
   end
+end
 
-  if %(development).include?(Rails.env)
-    Rake::Task["db:drop"].enhance(["search:destroy"])
-  end
+if %(development).include?(Rails.env)
```

Comment:
Should we fix this condition too? `%(development)` is a string literal and I don't think we wanted `String#include?` here. Probably best to add the `w`:

```ruby
%(development)
#=> "development"
%w(development)
#=> ["development"]
```

Reply:
Totally missed that in the refactoring. Fixed!